index.bs

```diff
@@ -3254,8 +3256,14 @@ credential.
         with more than one [=user account=] at the [=[RP]=].
 
     :   <dfn>displayName</dfn>
-    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display. For example, "Alex Müller" or "田中倫". The
-        [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice more than necessary.
+    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for
+        display. [=Clients=] may display this value to help users differentiate between multiple
+        [=credentials=] with the same value of {{PublicKeyCredentialEntity/name}}. Examples of
```

Comment:
Remove this line because name is the unique identifier typically.

Reply:
I believe I've addressed this, if you can take another look @akshayku I'd appreciate it.